daemon/src/model.rs

```diff
@@ -608,11 +608,11 @@ pub struct FundingRate(Decimal);
 impl FundingRate {
     pub fn new(rate: Decimal) -> Result<Self> {
         anyhow::ensure!(
```

Comment:
nit: it would be somehow clearer with a single check:

```rust
anyhow::ensure!(rate.abs() <= Decimal::ONE, "Funding rate can't be higher than 100%");
```